apps/demo-game/src/pages/admin/reports/[id].tsx

```
@@ -0,0 +1,458 @@
+
+  let totalDecisionAvg = [0, 0, 0]
+
+  let dataPerPeriod = []
+  for (let i = 0; i < numPeriods; i++) {
+    const playerResPerPeriod = previousSegmentResults.filter(
+      (result) => result.period.index === i
+    )
+    let dataPerPlayer = {}
+    playerResPerPeriod.map((result) => {
+      const decisions = Object.keys(result.facts.decisions).reduce((a, v) => {
+        return { ...a, [v]: Number(result.facts.decisions[v]) }
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)